validate/lab01_test.go

```diff
@@ -110,3 +104,10 @@ var _ = Describe("Lab 1 Containers", func() {
 		}
 	})
 })
+
+func YamlToInterface(path string) interface{} {
+	var output interface{}
+	file, _ := ioutil.ReadFile(path)
+	yaml.Unmarshal([]byte(file), &output)
```

Comment:
we should handle the possibility the the yaml completely fails to parse and yaml.Unmarshal returns an error

Reply:
Good point, I added some tests to handle both the file IO and unmarshalling